Packs/NCSCCyberAsssessmentFramework/IncidentTypes/incidenttype-NCSC_CAF_Assessment.json

```
@@ -0,0 +1,22 @@
+{
+    "autorun": true,
+    "color": "#32d296",
+    "days": 0,
+    "daysR": 0,
+    "default": false,
+    "disabled": false,
+    "hours": 0,
+    "hoursR": 0,
+    "id": "NCSC CAF Assessment",
+    "layout": "16efc0b7-517f-4492-8104-0896262ebe54",
+    "locked": false,
+    "name": "NCSC CAF Assessment",
+    "playbookId": "2fd1efe9-6f59-42e5-8db2-43a8799898e8",
```

Per my previous comment, please change the referenced id.
```suggestion
    "playbookId": "NCSC CAF Assessment",
```